src/Module.ts

```diff
@@ -244,6 +244,7 @@ export default class Module {
 	error(props: RollupError, pos: number) {
 		if (pos !== undefined) {
 			props.pos = pos;
+			props.graph = this.graph;
```

Comment:
Since `graph` is only being used to access `graph.watchFiles`, how about just passing those?

```suggestion
props.watchFiles = Object.keys(this.graph.watchFiles);
```

Reply:
👍👍 

Using `Object.keys` doesn't give you the watch files for some reason but the same `Array.from(this.graph.moduleById.keys())` does the trick. Changed in https://github.com/rollup/rollup/pull/3081/commits/7c003ac4bcd75bb44c850be9c8b46de6a56b4c1e